CONTRIBUTING.md

```diff
@@ -0,0 +1,108 @@
+# Contribution Guidelines
+
+Anyone is welcome to contribute to this project. Feel free to get in touch with other community
+members on [Matrix][1], the mailing list or through issues here on GitHub.
+
+## Code of Conduct
+
+You must agree to abide by the [Mozilla Community Participation Guidelines][2]
+
+## Bug Reports
+
+You can file issues here on GitHub. Please try to include as much information as you can and under
+what conditions you saw the issue.
+
+## Sending Pull Requests
+
+Patches should be submitted as pull requests (PR).
+
+Before submitting a PR:
+
+- Ensure you are pulling from the most recent `main` branch.
+- Your code must run and pass all the automated tests before you submit your PR for review.
+  "Work in progress" or "Draft" pull requests are allowed to be submitted, but should be clearly
+  labeled as such and  should not be merged until all tests pass and the code has been reviewed.
+- Your patch should include new tests that cover your changes. It is your and your reviewer's
+  responsibility to ensure your patch includes adequate tests.
+
+When submitting a PR:
+
+- You agree to license your code under the project's open source license ([MPL 2.0](/LICENSE)).
+- Base your branch off the current `main`.
+- Add both your code and new tests if relevant.
+- Your patch must be [GPG signed][3] to ensure the commits come from a trusted source.
```

Comment:
The `Your patch must be [GPG signed][3] to ensure the commits come from a trusted source.`

Reply:
🤷🏻‍♀️ I noticed this, or a version of this, is repeated in other repos, examples:
https://github.com/mozilla/fxa/blob/main/CONTRIBUTING.md#sending-pull-requests
https://github.com/mozilla-services/merino-py/blob/main/CONTRIBUTING.md#sending-pull-requests

Reply:
If you're not on the team you shouldn't be able to submit to the main repo, isn't that enough? To me that extra signing stuff is unnecessary.

Reply:
The signing seems standard for Mozilla no? but as i say that... is my key not working...

Reply:
I'd lean towards making sure everything is signed. 
I see a few mentions of "gpg" when I search the https://firefox-source-docs.mozilla.org/ docs.
But the reality is that it will almost certainly be only us four contributing to the repo. In the very unlikely chance we do get external contributors the likelihood of them reading the bullets is pretty low anyways -- and this is just documentation. Not sure we can enforce the signed-ness of commits elsewhere in the admin. 🤔

Reply:
Yes making it 'gpg' signed is the standard but we can let that go if we want for this repo.

Reply:
I'm with @Trinaa here. We enforce Git commit signing on many Mozilla owned repositories and that creates a more trust because it guarantees that the commits were authored by the person who's email address was used for the commit and that the commit contents were not altered later. 🔑 

I would however change the phrasing here so that Git commit signing is required, but it does not need to be GPG. For example GitHub supports Git commit signing using SSH: https://docs.github.com/en/authentication/managing-commit-signature-verification/telling-git-about-your-signing-key#telling-git-about-your-ssh-key

Reply:
I've worked at Mozilla for over 7 years, almost all on github, I've never had to sign commits till I started working on push. With 2 factor + SSH keys now, I think that's plenty. But looks like its a 3-2 vote, democracy.